ogre2/src/Ogre2Camera.cc

```diff
@@ -289,6 +289,34 @@ math::Matrix4d Ogre2Camera::ViewMatrix() const
   return Ogre2Conversions::Convert(this->ogreCamera->getViewMatrix(true));
 }
 
+//////////////////////////////////////////////////
+void Ogre2Camera::SetProjectionMatrix(const math::Matrix4d &_matrix)
+{
+  BaseCamera::SetProjectionMatrix(_matrix);
+  this->ogreCamera->setCustomProjectionMatrix(true,
+      Ogre2Conversions::Convert(this->projectionMatrix));
+}
+
+//////////////////////////////////////////////////
+void Ogre2Camera::SetProjectionType(CameraProjectionType _type)
+{
+  BaseCamera::SetProjectionType(_type);
+  if (this->projectionType == CPT_PERSPECTIVE)
```

Comment:
You could use a `switch` that has a default to perspective. Something like:
```
switch (this->projectionType)
{
  default:
  case CPT_PERSPECTIVE:
    this->ogreCamera->setProjectionType(Ogre::PT_PERSPECTIVE);
    break;
  case CPT_ORTHOGRAPHIC:
    this->ogreCamera->setProjectionType(Ogre::PT_ORTHOGRAPHIC);
    break;
}

Reply:
done. 4d7d3f7